Sources/Workspace/InitPackage.swift

```diff
@@ -873,9 +879,11 @@ public final class InitPackage {
         try makeDirectories(testModule)
 
         let testClassFile = try AbsolutePath(validating: "\(moduleName)Tests.swift", relativeTo: testModule)
+
         switch packageType {
-        case .empty, .buildToolPlugin, .commandPlugin, .executable, .tool: break
-        case .library:
+        case .empty, .buildToolPlugin, .commandPlugin:
+            break
+        case .library, .executable, .tool: // Added .executable and .tool here
```

Comment:
Similarly, this kind of comment isn't adding any useful information, and could become out of date over time.